ocs_ci/framework/main.py

```diff
@@ -66,6 +66,12 @@ def init_ocsci_conf(arguments=None):
     )
     parser.add_argument("--ocs-registry-image")
     parser.add_argument("--flexy-env-file", default="", help="Path to flexy env file")
+    parser.add_argument(
+        "--disable-components",
+        action="append",
+        choices=["rgw", "cephfs", "noobaa", "blockpools"],
+        help=("disable deployment of ocs components:rgw, cephfs, noobaa, blockpools."),
+    )
```

Comment:
Since it takes one _component_ peer argument, I would use singular in the argument name: `--disable-component`.
Also it will be good to mention in the help, that this argument can be used multiple times.